### PR TITLE
feat: [DHIS2-14577] search UX improvements

### PR DESCRIPTION
**Pull request by simonadomnisoru**
Small UX improvements to the search component. The program stage WL cypress tests failures are [known errors](https://github.com/dhis2/capture-app/wiki/Pull-Request-Guidelines#cypress-tests-to-follow-up-on-the-tests-are-commented-out-from-our-code-base) and will be fixed in https://github.com/dhis2/capture-app/pull/3269.

### CODE REVIEW

**Review by eirikhaugstulen — COMMENTED** [review]
Clean and simple 👍 One small comment, but nothing more.

File: src/core_modules/capture-core/components/SearchBox/SearchStatus/SearchStatus.component.js [hunk 1]
```diff
@@ -130,7 +131,29 @@ export const SearchStatusPlain = ({
             </Modal>
         );
     }
+
+    if (searchStatus === searchBoxStatus.UNIQUE_SEARCH_VALUE_EMPTY) {
+        return (
+            <Modal position="middle">
```
[the remark this answers is not in the document]
Comment (simonadomnisoru) [reply src/core_modules/capture-core/components/SearchBox/SearchStatus/SearchStatus.component.js]: Great catch! I updated the code with your suggestion. Thanks!

**Review by eirikhaugstulen — APPROVED** [review]
LGTM! 👍

**Review by geethaalwan — APPROVED** [review]
Tested successfully on 2.41,2.40.1,2.39.3,2.38.4 versions

**Pull request merged**